pdr_backend/lake/table.py

```diff
@@ -63,22 +55,15 @@ def save(self):
                 <= self.df.tail(1)["timestamp"].to_list()[0]
             )
 
-        filename = self._parquet_filename()
-
-        if os.path.exists(filename):  # "append" existing file
-            cur_df = pl.read_parquet(filename)
-            self.df = pl.concat([cur_df, self.df])
-
-            # drop duplicates
-            self.df = self.df.filter(pl.struct("ID").is_unique())
-            self.df.write_parquet(filename)
-            n_new = self.df.shape[0] - cur_df.shape[0]
-            print(f"  Just appended {n_new} df rows to file {filename}")
-        else:  # write new file
-            self.df.write_parquet(filename)
-            print(
-                f"  Just saved df with {self.df.shape[0]} rows to new file {filename}"
-            )
+        cur_df = self.csv_data_store.read_all(self.table_name, schema=self.df_schema)
+
+        self.df = pl.concat([cur_df, self.df])
+        self.df = self.df.filter(pl.struct("ID").is_unique())
+        self.csv_data_store.write(self.table_name, self.df, schema=self.df_schema)
+        n_new = self.df.shape[0] - cur_df.shape[0]
+        print(
+            f"  Just saved df with {n_new} df rows to the csv files of {self.table_name}"
+        )
```

Comment:
**[Load from + Write to CSVs...Just append]**
We are not doing the same patterns with CSVs that we were doing with parquet.
This code will re-save everything using CSVs every-time.

Instead:
- [ ] Please create a new function called `append`
- [ ] After fetching new data, users will append data to a table.
- [ ] Inside of append(), call a new function called `append_to_csv`. We just want to append a few rows at at time to a csv with as little latency as possible.
- [ ] Inside of append(), call a new function called `append_to_duckdb`. Comment this out. Address this in the next ticket.

Reply:
I have updated the DoD of the task, please update there.